addons/mail/static/src/new/public/boot.js

```diff
@@ -33,6 +37,7 @@ import { DiscussPublic } from "./discuss_public";
     // });
     const env = makeEnv();
     await startServices(env);
+    mapLegacyEnvToWowlEnv(Component.env, env);
```

Comment:
Do you know for which feature we need to add the legacy env again?
Ideally we should be able to work without it on the new public page.

Reply:
We need this for the tour on the public page, the tour relies on the `env.isDebug`